src/actions/MatrixActionCreators.js

```diff
@@ -37,11 +36,10 @@ function createSyncAction(matrixClient, state, prevState) {
 }
 
 /**
- * An action creator that will map an account data matrix event to a
- * MatrixActions.accountData action.
+ * Create a MatrixActions.accountData action that represents a MatrixClient `accountData`
+ * matrix event.
  *
- * @param {MatrixClient} matrixClient the matrix client with which to
- *                                    register a listener.
+ * @param {MatrixClient} matrixClient the matrix client.
```

Comment:
why do we pass this in, given it's ignored?

Reply:
Basically so that we can handle both action creators the same in `_addMatrixClientListener`